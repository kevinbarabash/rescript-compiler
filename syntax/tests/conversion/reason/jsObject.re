let component = props##"Component"

let element = props##element

let y = {"age": 30}
let y = {"age": 30, "name": "steve"}
