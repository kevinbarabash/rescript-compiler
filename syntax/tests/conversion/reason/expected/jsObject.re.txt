let component = props["Component"]

let element = props["element"]

let y = {"age": 30}
let y = {"age": 30, "name": "steve"}

type propField<'a> = {.}
type propField<'a> = {..} as 'a
type propField<'a> = {..} as 'a
type propField<'a> = Js.nullable<{..} as 'a>

type propField<'a> = {"a": b}
type propField<'a> = {.."a": b}
type propField<'a> = {"a": {"b": c}}
